.github/workflows/check.yml

```diff
@@ -109,20 +109,17 @@ jobs:
 
       - name: Build
         run: |
-          cargo +${{ matrix.rust-toolchain }} build --all-targets
+          cargo +${{ matrix.rust-toolchain }} build --all-targets --features ci,bench
           echo "LD_LIBRARY_PATH=${{ github.workspace }}/dist/Debug/lib" >> "$GITHUB_ENV"
           echo "DYLD_FALLBACK_LIBRARY_PATH=${{ github.workspace }}/dist/Debug/lib" >> "$GITHUB_ENV"
           echo "${{ github.workspace }}/dist/Debug/lib" >> "$GITHUB_PATH"
 
       - name: Run tests and determine coverage
-        run: cargo +${{ matrix.rust-toolchain }} llvm-cov nextest --features ci --all-targets --no-fail-fast --lcov --output-path lcov.info
-
-      - name: Benches should at least build
-        run: cargo +${{ matrix.rust-toolchain }} build --features bench --benches
+        run: cargo +${{ matrix.rust-toolchain }} llvm-cov nextest \
+           --all-targets --features ci,bench --no-fail-fast --lcov --output-path lcov.info
 
       - name: Run client/server transfer
         run: |
-          cargo +${{ matrix.rust-toolchain }} build --bin neqo-client --bin neqo-server
```

Comment:
Removing this seems to make the benchmark hang in some cases.

Reply:
Oh, that's because what happens is that the build needs to run for these targets.  We should probably add `--feature ci,bench` here so that it can use the previous build.